### ISSUE TITLE
[BUG] Voiceover doesn't work correctly for UiMulitpleChoices component

### ISSUE THREAD

**Issue by mateuszkulpa** [infermedica/component-library#218]
Voiceover doesn't read the legend of the fieldset.
Issue appears on chromium and firefox browser.

Current result

https://github.com/infermedica/component-library/assets/17556031/8e56b707-f492-4824-8690-127613085ac7

Expected result:

https://github.com/infermedica/component-library/assets/17556031/cb901f1f-bd41-4f9b-bd40-86ad2d936472
